src/routes/changelog/(entries)/2025-02-06.markdoc

```diff
@@ -0,0 +1,22 @@
+---
+layout: changelog
+title: "Appwrite 1.6.1 is now available"
+date: 2025-02-06
+cover: /images/changelog/2025-02-06.png
+---
+
+We're excited to announce a new update packed with features, fixes, and improvements to enhance your self-hosted Appwrite instance! This release focuses on performance, stability, and expanded functionality.
+
+Most notably, this release includes:
+
+* Improved compatibility for webp requests
```

Comment:
A lot has changed between 1.6.0 and 1.6.1 and the changelog doesn't cover them. It should be more detailed IMO

Reply:
I added a few more bullet points. Developers should be able to go into the github release to see the full details.